gemrb/core/Interface.cpp

```diff
@@ -3563,10 +3563,13 @@ void Interface::SanitizeItem(CREItem *item) const
 	// some slot flags might be affected by the item flags
 	if (!(item->Flags & IE_INV_ITEM_CRITICAL)) {
 		item->Flags |= IE_INV_ITEM_DESTRUCTIBLE;
-	} else if ((core->HasFeature(GFFlags::SELLABLE_CRITS_NO_CONV)) && (!(item->Flags & IE_INV_ITEM_CONVERSABLE))) {
+	} else if (core->HasFeature(GFFlags::SELLABLE_CRITS_NO_CONV)) {
 		item->Flags |= IE_INV_ITEM_DESTRUCTIBLE;
-	}
-
+		}
+	if  (item->Flags & IE_INV_ITEM_CONVERSABLE) {
```

Comment:
This needs to happen only in iwd, unless you have some evidence items with the conversable bit shouldn't be destructible.

Reply:
And to quote the expected behaviour:
> All items survive effects like disintegration and freezing/stone death irrespective of bit 0

IOW no items should have the bit set.

Reply:
I couldn't find any items with bit 11 set in vanilla totsc, so iwd appears to be the only one using it despite the comment in the ITM docs.  I have yet to test/compare with bg2.

From the behaviour of the original games a 'critical' bit should permit selling in both bg1 and iwd/how, but the 'conversable' bit only appears to be used as unsellable in iwd/how.  

Should this be two separate flags as well or stick with one flag?

Reply:
Isn't this still the same difference, already documented in the initial report? 

I can't think of any users of bit 11 outside of bg2 (lilarcor) off the top of my head. The chatty sword in iwd you can't interact with and I'm not sure if pst needs it, since it has a whole field for the dialog name.